scenario_configs/eval5/carla_video_tracking/carla_video_tracking_goturn_advtextures_defended.json

```diff
@@ -14,11 +14,11 @@
         "use_label": true
     },
     "dataset": {
-        "batch_size": 1,
-        "eval_split": "dev",
-        "framework": "numpy",
-        "module": "armory.data.adversarial_datasets",
-        "name": "carla_video_tracking_dev"
+        "test": {
```

Comment:
did you mean to change this from `dev` to `test`?

Reply:
No this should be `dev`, I changed it to `test` for testing. Will fix in my next push.

Reply:
Addressed by d9862ce7a7409946eb90f9a513a6df4150871adc.